packages/pyinstaller/PKGBUILD

```
@@ -2,41 +2,72 @@
 # See COPYING for license details.
 
 pkgname=pyinstaller
-_pkgname=PyInstaller
-pkgver=3.6
-pkgrel=4
+pkgver=6.11.1
+pkgrel=1
 epoch=2
 groups=('blackarch' 'blackarch-misc')
-pkgdesc='A program that converts (packages) Python programs into stand-alone executables, under Windows, Linux, Mac OS X, Solaris and AIX.'
-arch=('any')
+pkgdesc='Bundles a Python application and all its dependencies into a single package.'
+arch=('x86_64' 'aarch64')
 url='http://www.pyinstaller.org/downloads.html'
```

```suggestion
url='https://pyinstaller.org'
```
The `downloads.html` page does not exist anymore.